package indigo.pb;

import "sender_message.proto";

option java_package = "ru.indigosystem.serv.pb";

option java_outer_classname = "Skel2";
option optimize_for = LITE_RUNTIME;

message skel_message2 {
	extend indigo_msg {
                optional skel_message2 skel_message_2_out = 201;
	}

	extend internal_msg {
                optional skel_message2 skel_message_2_in = 201;
	}

	optional int32 data2 = 2 [ default = 2 ];
}

